vpr/src/route/router_delay_profiling.cpp

```diff
@@ -124,7 +123,7 @@ std::vector<float> calculate_all_path_delays_from_rr_node(int src_rr_node, const
         device_ctx.rr_nodes,
         &g_vpr_ctx.device().rr_graph,
         device_ctx.rr_rc_data,
-        device_ctx.rr_switch_inf,
+        g_vpr_ctx.mutable_device().rr_graph_builder.rr_switch_inf(),
```

Comment:
Should not use a mutable object. A profiling function is not allowed to modify any routing resource graph.